src/acquisition/covidcast/database.py

```diff
@@ -570,39 +581,38 @@ def get_covidcast_meta(self):
               `geo_value`
             FROM
               `covidcast`
+            WHERE
+              `source` = %s AND
+              `signal` = %s
             GROUP BY
               `time_value`,
               `time_type`,
               `geo_type`,
-              `source`,
-              `signal`,
               `geo_value`
           ) x
         ON
           x.`max_issue` = t.`issue` AND
           x.`time_type` = t.`time_type` AND
           x.`time_value` = t.`time_value` AND
-          x.`source` = t.`source` AND
-          x.`signal` = t.`signal` AND
           x.`geo_type` = t.`geo_type` AND
           x.`geo_value` = t.`geo_value`
       WHERE
-        t.`signal` NOT LIKE 'wip_%'
+        NOT t.`is_wip` AND
+        t.`source` = %s AND
+        t.`signal` = %s
```

Comment:
I suggest to keep merging on `source`, and `signal` in order to avoid filtering by them twice.

Reply:
i didnt like seeing the repetition, but thats a good point